apps/zotonic_mod_authentication/priv/templates/sessions.tpl

```diff
@@ -0,0 +1,139 @@
+                    <strong>#{{ m.acl.user.id }} ({{ m.acl.user.id.title }})</strong><br>
+                    {_ Note: sessions will appear here only when actively used by a browser tab, it's recommended to keep this page open for a while. _}
+                </p>
+
+                <table id="active_sessions" class="table">
+                    <thead>
+                        <tr>
+                            <th scope="col">{_ Session ID _}</th>
+                        </tr>
+                    </thead>
+
+                    <tbody id="active_sessions_rows">
+                    </tbody>
+                </table>
+                {% button class="btn btn-danger" text=_"Interrupt all active sessions"
+                    action={
+                        dialog_open
+                        title=_"Are you sure?"
+                        template="_dialog_session_close.tpl"
+                    }
+                %}
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="row">
+    <div class="col-md-12 col-lg-12 col-sm-12">
+        <div class="panel panel-info">
+            <div class="panel-heading">
+                <h3 class="panel-title">{_ Last known logons _}</h3>
+            </div>
+            <div class="panel-body">
+
+                <p class="text-muted">
+                    {_ Below are the last known logons from your user account. _}<br>
+                    {_ Note: these are periodically deleted. _}
+                </p>
+
+                <table id="known_logons" class="table">
+                    <thead>
+                        <tr>
+                            <th scope="col">#</th>
+                            <th scope="col">{_ User Agent _}</th>
+                            <th scope="col">{_ IP Address _}</th>
+                            <th scope="col">{_ Logon time _}</th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        {% for user_agent, ip_address, logon_time in m.identity[m.acl.user.id].logon_history %}
+                            <tr>
+                                <th scope="row">{{ forloop.counter }}</th>
+                                <td>{{ user_agent }}</td>
+                                <td>{{ ip_address }}</td>
+                                <td>{{ logon_time|date:_"d M Y, H:i" }}</td>
+                            </tr>
+                        {% empty %}
+                            <tr>
+                                <th scope="row">{_ No recent known logons _}</th>
+                                <td>-</td>
+                                <td>-</td>
+                                <td>-</td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+</div>
+
+{% javascript %}
+cotonic.broker.subscribe("bridge/origin/test/{{ m.acl.user.id }}/session_extended", function(msg) {
```

Comment:
Better use the topic:

``user/:user_id/session_extended``

As the `test` topic is public and then everybody can listen in.
The `user/:user_id/...` topic is restricted to the user or those that can _edit_ the user.

Reply:
Forgot to change it from `test`, switched to `user` in c52ab1f2abfec75e16bcc7895ed8abcab04d010e